testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -193,10 +269,17 @@ public static Credential getCredentials(LabUserQuery query) {
     }
 
     public static AppInfo getAppInfo() {
+            return getAppInfo(UserType.CLOUD, AzureEnvironment.AZURE_CLOUD, SignInAudience.AZURE_AD_MULTIPLE_ORGS,
+                    IsAdminConsented.YES, PublicClient.YES);
+   }
```

Comment:
Do we really need this method? Is the one below not sufficient?

Reply:
This is just convenience, since these are supposed to be the "default" values according to the swagger document.  I could theoretically just send a to Api/App and get these, and that's what I should probably change this to do.

Reply:
Okay I would suggest renaming the method then to something like `getDefaultAppInfo`

Reply:
Yeah, it's interesting the mismatch between what the service does and what the AppApi class allows.